# SOME DESCRIPTIVE TITLE.
# Copyright (C) 2019, Qiskit Development Team
# This file is distributed under the same license as the Qiskit package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: Qiskit \n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-06-03 20:57+0000\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: ../../../autodoc/qiskit.providers.ibmq.circuits.exceptions.rst:2
msgid "qiskit.providers.ibmq.circuits.exceptions module"
msgstr ""

#: ../../../../../../../virtualenv/python3.7.1/lib/python3.7/site-packages/qiskit/providers/ibmq/circuits/exceptions.py:docstring of qiskit.providers.ibmq.circuits.exceptions:1
msgid "Exceptions related to Circuits."
msgstr ""

#: ../../../../../../../virtualenv/python3.7.1/lib/python3.7/site-packages/qiskit/providers/ibmq/circuits/exceptions.py:docstring of qiskit.providers.ibmq.circuits.exceptions.CircuitAvailabilityError:1
#: ../../../../../../../virtualenv/python3.7.1/lib/python3.7/site-packages/qiskit/providers/ibmq/circuits/exceptions.py:docstring of qiskit.providers.ibmq.circuits.exceptions.CircuitResultError:1
#: ../../../../../../../virtualenv/python3.7.1/lib/python3.7/site-packages/qiskit/providers/ibmq/circuits/exceptions.py:docstring of qiskit.providers.ibmq.circuits.exceptions.CircuitSubmitError:1
msgid "ベースクラス: :class:`qiskit.providers.ibmq.circuits.exceptions.CircuitError`"
msgstr ""

#: ../../../../../../../virtualenv/python3.7.1/lib/python3.7/site-packages/qiskit/providers/ibmq/circuits/exceptions.py:docstring of qiskit.providers.ibmq.circuits.exceptions.CircuitAvailabilityError:1
msgid "Error while accessing a Circuit."
msgstr ""

#: ../../../../../../../virtualenv/python3.7.1/lib/python3.7/site-packages/qiskit/providers/ibmq/circuits/exceptions.py:docstring of qiskit.providers.ibmq.circuits.exceptions.CircuitError:1
msgid "ベースクラス: :class:`qiskit.providers.ibmq.exceptions.IBMQError`"
msgstr ""

#: ../../../../../../../virtualenv/python3.7.1/lib/python3.7/site-packages/qiskit/providers/ibmq/circuits/exceptions.py:docstring of qiskit.providers.ibmq.circuits.exceptions.CircuitError:1
msgid "Generic Circuit exception."
msgstr ""

#: ../../../../../../../virtualenv/python3.7.1/lib/python3.7/site-packages/qiskit/providers/ibmq/circuits/exceptions.py:docstring of qiskit.providers.ibmq.circuits.exceptions.CircuitResultError:1
msgid "Error during the results of a Circuit."
msgstr ""

#: ../../../../../../../virtualenv/python3.7.1/lib/python3.7/site-packages/qiskit/providers/ibmq/circuits/exceptions.py:docstring of qiskit.providers.ibmq.circuits.exceptions.CircuitSubmitError:1
msgid "Error while submitting a Circuit."
msgstr ""

